AspNetCore.Diagnostics.HealthChecks.sln

```diff
@@ -240,8 +240,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "workflows", "workflows", "{
 		.github\workflows\healthchecks_aws_s3_cd_preview.yml = .github\workflows\healthchecks_aws_s3_cd_preview.yml
 		.github\workflows\healthchecks_aws_s3_ci.yml = .github\workflows\healthchecks_aws_s3_ci.yml
 		.github\workflows\healthchecks_aws_secretsmanager_cd.yml = .github\workflows\healthchecks_aws_secretsmanager_cd.yml
-		.github\workflows\healthchecks_aws_secretsmanager_cd_preview.yml = .github\workflows\healthchecks_aws_secretsmanager_cd_preview.yml
```

Comment:
Why?

Reply:
Is that cd_preview workflow used? Quite didn't understand why I had to delete the secrets one, and not the systems one...And on preview PR was that left out without using that. So is that cd_preview workflow even used at all on any of those features?

Reply:
> Quite didn't understand why I had to delete the secrets one

Had to delete?

Reply:
My bad. I misunderstood :-(